web/html/src/manager/virtualization/guests/console/MessagePopUp.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { PopUp } from 'components/popup';
+import { Loading } from 'components/utils/Loading';
+import { Form } from 'components/input/Form';
+import { Password } from 'components/input/Password';
+import { Button } from 'components/buttons';
+import { hideDialog } from 'components/dialog/util';
+
+type Props = {
+  id: string,
+  onSubmit?: () => void,
+  popupState: string,
```

Comment:
Could possibly use either an enum here or a fixed string constant a-la `popupState: "wait" | "askPassword" | "errors"` for clarity.

Reply:
yep, will do. thanks for the hint